### PR TITLE
Check in/71440/add 45min reminder to ga

### PR DESCRIPTION
**Pull request by wullaski**
## Summary

- Adds 45MR context to GA events

There was a request in the ticket to add to validations calls to the /session endpoint but we don't have a way of knowing if it is a 45 minute reminder session until we get the check in data.

## Related issue(s)

- [department-of-veterans-affairs/va.gov-team#71440](https://github.com/department-of-veterans-affairs/va.gov-team/issues/71440)

## Testing done

- Tested events with adswerve

## What areas of the site does it impact?

GA events for:
- Demographics
- Travel
- Check-in Success
- Check-in Failure

## Acceptance criteria

- [ ] Adds the string 45MR to events when `setECheckinStartedCalled` is true

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### CODE REVIEW

File: src/applications/check-in/api/versions/v2.js [hunk 1]
```diff
@@ -6,15 +6,21 @@ import environment from '@department-of-veterans-affairs/platform-utilities/envi
...
+  }) => {
     const url = '/check_in/v2/sessions/';
     let requestUrl = `${environment.API_URL}${url}${token}`;
     if (checkInType) {
       requestUrl = appendQuery(requestUrl, {
         checkInType,
       });
     }
-    const eventLabel = `${checkInType || 'day-of'}-get-current-session-dob`;
+    const eventLabel = `${checkInType || 'day-of'}-get-current-session-dob${
```
Comment (lee-delarm6) [inline src/applications/check-in/api/versions/v2.js:21]: Just checking, but this is something like -session-dob45 and not meant to be -session-dob-45 right?

File: src/applications/check-in/api/versions/v2.js [hunk 2]
```diff
@@ -96,7 +102,7 @@ const v2 = {
 
     const json = await makeApiCallWithSentry(
       apiRequest(`${environment.API_URL}${url}`, settings),
-      'check-in-user',
+      `check-in-user${setECheckinStartedCalled ? '-15MR' : ''}`,
```
Comment (lee-delarm6) [inline src/applications/check-in/api/versions/v2.js:105]: See above comment, -15MR vs 15MR

**Review by lee-delarm6 — CHANGES_REQUESTED** [review]
If these are intended, let me know and I'll switch to approve!

File: src/applications/check-in/api/versions/v2.js (hunk 1 above)
Comment (wullaski) [reply src/applications/check-in/api/versions/v2.js]: Whoops! I meant to remove that since it's not possible to pass that parameter into the call to the session endpoint.

**Review by lee-delarm6 — DISMISSED** [review]
Looks good now :)

**Comment by brianseek** [comment]
Shouldn't these all be 45MR and not 15MR?

File: src/applications/check-in/api/versions/v2.js [hunk 3]
```diff
@@ -6,7 +6,7 @@ import environment from '@department-of-veterans-affairs/platform-utilities/envi
 import { makeApiCallWithSentry } from '../utils';
 
 const v2 = {
-  getSession: async ({ token, checkInType }) => {
+  getSession: async ({ token, checkInType = 'day-of' }) => {
```
Comment (brianseek) [inline src/applications/check-in/api/versions/v2.js:9]: Why this change? If you change this here, we would need changes down in the code too. But I think this was purposely built this way to not add the query string for day-of.

**Review by brianseek — CHANGES_REQUESTED** [review]
I feel like its supposed to be 45MR not 15MR and one comment on the change to getSession

**Comment by wullaski** [comment]
At some point it changed to 15MR in my head and I made that mistake everywhere 😫 good catch.

File: src/applications/check-in/api/versions/v2.js (hunk 3 above)
Comment (wullaski) [reply src/applications/check-in/api/versions/v2.js]: I meant to remove this as well thanks for catching it.

**Review by lee-delarm6 — APPROVED** [review]
Huzzah, 45MR everywhere now!

**Review by ataker — APPROVED** [review]

**Review by brianseek — APPROVED** [review]
LGTM
